configuration/README.md

```diff
@@ -59,3 +59,12 @@ To support execution in a simulated environment, V2X-Hub is in the process of in
 * **V2XHUB_IP** – Environment variable for storing IP address of V2X Hub.
 * **INFRASTRUCTURE_ID** – Environment variable for storing infrastructure id of V2X Hub.
 * **SENSOR_JSON_FILE_PATH** – Environment variable for storing path to sensor configuration file. This is an optional simulation environment variable that allows for setting up simulated sensor for a V2X-Hub instance. Example file can be found in the **CDASimAdapterPlugin** tests [here](../src/v2i-hub/CDASimAdapter/test/sensors.json).
+
+### Open V2X-Hub in Google Chrome 
+If the chromium browser is not available, run this command to open V2X-Hub in google chrome:
```

Comment:
To access V2X-Hub UI please use either chromium or google-chrome browser  and run the following command 
```bash
chromium --ignore-certificate-errors <v2xhub_ip>
```
or 
```bash
google-chrome --ignore-certificate-errors <v2xhub_ip>
```

Reply:
Updated the README file.